### PR TITLE
ICU-21248 Adds line ending check to Travis CI check. The eol check fits

### PR DESCRIPTION
**Pull request by gnrunge**
quite well into what the script is doing already.
Note that one file, 'testrunner.cmd', contains \r, which seems to
be acceptable since it is a Windows file.

##### Checklist

- [ ] Issue filed: https://unicode-org.atlassian.net/browse/ICU-21248
- [ ] Updated PR title and link in previous line to include Issue number
- [ ] Issue accepted
- [ ] Tests included
- [ ] Documentation is changed or added

### CODE REVIEW

File: tools/scripts/icu-file-utf8-check.py [hunk 1]
```diff
@@ -38,14 +38,18 @@
 # all of icu/. Modify as needed.
 icu_directories_to_be_scanned = ["."]
 
+# Files that are allowed to contain \r line endings. If this list
+# grows to long consider a file instead.
```
Comment (markusicu) [inline tools/scripts/icu-file-utf8-check.py:42]: to --> too

File: tools/scripts/icu-file-utf8-check.py [hunk 2]
```diff
@@ -116,6 +124,15 @@ def main(argv):
             if check_file(f, source_file) != 0:
                 exit_status = 1
 
+            # Lastly, check the line endings of the file.
+            # Not that 'grep' returns null if it reports a file,
```
Comment (markusicu) [inline tools/scripts/icu-file-utf8-check.py:128]: Not --> Note

**Review by markusicu — COMMENTED** [review]
What does "ICU-2128 Remove superflous import." refer to?

Please remove that from both the commit message and the PR description.

File: tools/scripts/icu-file-utf8-check.py (hunk 2 above)
Comment (gnrunge) [reply tools/scripts/icu-file-utf8-check.py:128]: "... superflous import ..." refers to a quick second commit (stashed) after I realized the script still has 'import subprocess' from some debug work ...

File: tools/scripts/icu-file-utf8-check.py (hunk 1 above)
Comment (gnrunge) [reply tools/scripts/icu-file-utf8-check.py]: Done

File: tools/scripts/icu-file-utf8-check.py (hunk 2 above)
Comment (gnrunge) [reply tools/scripts/icu-file-utf8-check.py]: Done

**Comment by markusicu** [comment]
Please do remove this trailing part from the commit message (via `git commit --amend`). The first line anyway has a typo in the ticket number, and the second line is just noise.
```
ICU-2128 Remove superflous import.

ICU-21248 Two small corrections (typos) in response to review comments.
```

**Comment by gnrunge** [comment]
Commit message updated.

**Review by srl295 — APPROVED** [review]
LGTM and commit message seems fixed

**Review by markusicu — APPROVED** [review]
lgtm tnx

**Comment by markusicu** [comment]
Merged on Norbert's behalf because he is off for a few days.

**Pull request merged**